src/types/object-node.ts

```diff
@@ -199,6 +199,12 @@ export function createObjectFactory(arg1, arg2?) {
     )
 }
 
+function getObjectFactoryBaseModel(item){
```

Comment:
@mweststrate Here I get the ObjectFactory baseModel and silently fallback to an empty object for items that are'nt object factories. Let me know if you thought about another behaviour.